components/brave_wallet/browser/polkadot/polkadot_wallet_service.cc

```diff
@@ -38,4 +39,14 @@ void PolkadotWalletService::GetNetworkName(mojom::AccountIdPtr account_id,
                                        std::move(callback));
 }
 
+void PolkadotWalletService::GetAccountBalance(
```

Comment:
I think chain_id should be given as a method argument, similar way to the Eth, to be able to fetch different balances for Substrate networks